syntax = "proto3";
 package edu.itu.cs.atcas.infrastructure.ml;

// Response message
message AirTrafficResponse {
  repeated double sector_density = 1;
  repeated double loss_of_separation = 2;
  repeated double speed_deviation = 3;
  repeated double airflow_complexity = 4;
  repeated double sector_entry = 5;
}

// Request message
message AirTrafficRequest {
  string configuration_id = 1;
}

// Service definition
service AirTrafficService {
  rpc GetAirTrafficInfo (AirTrafficRequest) returns (AirTrafficResponse);
}